exercises/proverb/.meta/src/reference/java/Proverb.java

```diff
@@ -0,0 +1,23 @@
+final class Proverb {
+    private String[] words;
+
+    Proverb(String[] words) {
+        this.words = words;
+    }
+
+    String recite() {
+        if (words.length < 1) return "";
+        String initWord = "",
```

Comment:
Could this just be set to `words[0]` instead of setting it to an empty string and then immediately checking if it's empty?